devops/kubernetes/cluster/index.ts

```diff
@@ -0,0 +1,37 @@
+import * as awsx from '@pulumi/awsx'
+import * as aws from '@pulumi/aws'
+import * as eks from '@pulumi/eks'
+import * as k8s from '@pulumi/kubernetes'
+import * as pulumi from '@pulumi/pulumi'
+
+const awsConfig = new pulumi.Config('aws')
+const config = new pulumi.Config()
+
+const numberOfAvailabilityZones = parseInt(config.get('numberOfAvailabilityZones') || '2')
+const clusterName = config.get('clusterName') || 'eksctl-joystream'
+const instanceType = (config.get('instanceType') || 't2.medium') as pulumi.Input<aws.ec2.InstanceType>
+
+export let kubeconfig: pulumi.Output<any>
+export let provider: k8s.Provider
+export let platform = config.require('platform')
+
+if (platform === 'minikube') {
```

Comment:
Looking back at these options and trying things out, it doesn't seem necessary to me to have to create a pulumi stack when the provider is just going to be "local" cluster. So I suggest we drop the minikube option, and this script should only deploy clusters to aws, and later maybe add support for google and azure.